src/query/frontend/opencypher/grammar/MemgraphCypher.g4

```diff
@@ -32,46 +32,52 @@ memgraphCypherKeyword : cypherKeyword
                       | BATCH_LIMIT
                       | BATCH_SIZE
                       | BEFORE
-                      | BOOTSTRAP_SERVERS
                       | BUILD
+                      | BOOTSTRAP_SERVERS
```

Comment:
Was this move intentional?

Reply:
No, shouldn't make a difference. I just copied the keywords from the lexer file and that was the order there.